Source/WarpX.cpp

```diff
@@ -1303,11 +1306,6 @@ WarpX::ReadParameters ()
             "Vay current deposition not implemented for Galilean algorithms"
         );
 
-        WARPX_ALWAYS_ASSERT_WITH_MESSAGE(
-            (!current_correction) || v_galilean_is_zero || (!fft_do_time_averaging),
-            "Current correction not implemented for averaged Galilean algorithm"
-        );
```

Comment:
I don't see obvious reasons why we should not be allowed to use the Galilean current correction with time averaging. Maybe to be discussed offline.

Reply:
Hm, I agree we could remove this.

I seems that in general, the current correction is not implemented for the case where `J` is linear in time:
https://github.com/ECP-WarpX/WarpX/blob/development/Source/FieldSolver/SpectralSolver/SpectralAlgorithms/PsatdAlgorithmJLinearInTime.cpp#L427
But we already have an abort there, so no need to have another check in `WarpX.cpp`

Reply:
Implemented in https://github.com/ECP-WarpX/WarpX/pull/3092/commits/ce22f93a6dd437a1daaede9dee2afd3d69fdabba (default changed to `False` for multi-J).